common/nymsphinx/src/addressing/nodes.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use sphinx::constants::NODE_ADDRESS_LENGTH;
+pub use sphinx::constants::NODE_ADDRESS_LENGTH;
```

Comment:
I need it in order to determine what length an address should be so that I can accomplish the gateway routing we talked about yesterday. Is there a different constant that would be better to depend on?

Reply:
This is a re-export to avoid having the gateway depend directly on Sphinx.